src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -211,17 +211,19 @@ gl_sframe augment_data(const gl_sframe &data,
         // write the synthetically generated image and the constructed
         // annotation to output SFrame.
         output_writer.write({synthetic_image, annotation}, segment_id);
-        augmented_counter.fetch_add(1, std::memory_order_relaxed);
+        size_t augmented_rows_completed = augmented_counter.fetch_add(1);
```

Comment:
Hmm, what does this print in your testing?

Upon closer inspection, `fetch_add` returns the value *before* the addition, so you might need to add one here

Reply:
It was printing a 0 that I should have thought of as anomalous earlier. Sorry about that. Fixed.